assets/front/scss/0_1_base/_mixins.scss

```diff
@@ -470,30 +470,36 @@
 
 // Custom mixins
 
-// Rows CSS based
-// This will be not needed anymore with the flex :D
-@mixin make-grid-smart-rows( $items: $grid-columns, $offset : 0, $breakpoints: $grid-breakpoints ) {
-  @each $breakpoint in map-keys($breakpoints) {
+//RTL offset-* classes
+//inspired by the bootstrap offset-* class generator, used to make offset-* classes rtl compliant
+@mixin make-col-offset-rtl($size, $columns: $grid-columns) {
+  $num: $size / $columns;
+  margin-right: if($num == 0, 0, percentage($num));
+  margin-left: auto;
+}
 
```

Comment:
cool tnx

Reply:
that part is copied from bs :P 
I just replaced margin-left with margin-right and added margin-left: auto; to override the default offset-* :D

Reply:
who said that copying ( and adapting ) good open source code was bad ?... ;) :D

Reply:
Nobody :D Just didn't want to take different credits besides the ones that I already deserve, they're already a lot :P

Reply:
ah ah ah
well, I agree :)